### ISSUE TITLE
import tmdb watchlist

### ISSUE THREAD

**Issue by reitenth** [sbondCo/Watcharr#575]
ive imported ratings, but watchlist just imports to finished. thanks

**Comment by IRHM** [comment]
Hi @reitenth, I think that sounds right, was something else expected?

**Comment by reitenth** [comment]
yes, it would be more useful if watchlist import for tmdb imported to "planned" instead of "finished"
